### ISSUE TITLE
Patching is not working on Windows

### ISSUE THREAD

**Issue by jmelosegui** [terrajobst/git-istage#47]
I have updated to the latest version of the tool, and now I cannot use git istage. The tool is able to detect the changes, but I cannot stage any line or block, nor can I un-stage them.

![istage-patch-error](https://user-images.githubusercontent.com/450246/145685656-2e8486d6-141a-41bc-a1bc-429528e2ba1c.gif)

Debugging the tool (using `git bisect` 😉), I found the issue started manifesting after PR #43.
One of the updates was [changing the way the patch file is created](https://github.com/terrajobst/git-istage/commit/f27cda35d1200c4ab9af6b41297dfc2afdbbc811#diff-1a617748b81e6fccd6757d7fe88df32102377686d1374c21ed8cf0a0887d0518L145-R145). It went from using `File.WriteAllText` to use `File.WriteAllLines`.

Then I compared the differences between these two methods (see image below), and I found that the latter introduces a `CRLF` instead of keeping the original `LF` character. So seems like in Windows OS, the method `File.WriteAllLines` will add that extra `CR` (`0x0D`) per line. This will result in a conflict when applying the patch. 

![image](https://user-images.githubusercontent.com/450246/145686157-8665b46b-ecc1-4d69-8ce6-4d465d49f5d1.png)

**Comment by DualBrain** [comment]
Is there any sort of work-around to this?  I see it was reported about 9 mo ago and there does appear to be a PR; but since that hasn't been merged and a new version made available... what can someone do to get this working?  Is it possible to install an older release that does work?  If so, which version... and how does one go about installing this not using the existing instructions of `dotnet tool install git-istage -g` or is this even possible?

**Comment by jmelosegui** [comment]
@DualBrain, while I wait for @terrajobst to have some time to review this, I am using a nightly build version of the tool hosted on my own feed.

If you want to use it, run the following command on your computer.

```
dotnet tool update -g git-istage --version 0.3.59-g64a4c63489 --add-source https://f.feedz.io/jmelosegui/oss/nuget/index.json
```

Hope this helps.

**Comment by terrajobst** [comment]
Sorry for the long delay; it’s now fixed and in 8c6bce5b576f297fd238e9685c26f2ba5592e758.
